linux/Dockerfile

```diff
@@ -19,6 +21,38 @@ RUN apt-get update && \
 	curl \
 	jq
 
+# --- kcov --- #
+# Install dependencies
+RUN apt-get install -y \
```

Comment:
This `RUN` should be combined with above one that already has `apt-get install`. No need for extra layers here.

Additionally to avoid storing apt cache in the image, i'd add `&& apt-get clean`. As the cache is created and removed in the same step, it will not hang around in the intermediate layers.